jobs/build/release/Jenkinsfile

```diff
@@ -207,6 +207,24 @@ node {
                 }
             }
 
+            stage("request upgrade tests") {
```

Comment:
Why wait until everything else is done? They should start upgrade tests as soon as the release exists, i.e. right before `stage("wait for stable")`

Reply:
Great insight. Next push.